lib/access_import/journal_transform.rb

```diff
@@ -0,0 +1,35 @@
+require 'acc_utils'
+
+class JournalTransform
+  include AccUtils
+
+  def initialize(*accessors)
+    accessors.each do |accessor|
+      instance_variable_set("@#{accessor.class.name.underscore}", accessor)
+    end
+  end
+
+  def prepare_attributes(journal, person, assignment, user_id)
+    {
+      body: journal[:m_Text],
+      journalable_type: person.class.name,
+      journalable_id: person.id,
+      assignment_id: assignment&.id,
+      created_at: journal[:d_ErfDatum],
+      updated_at: journal[:d_MutDatum],
```

Comment:
Not sure if Rails will overwrite these, or how important it is to keep the original dates